app/views/client/softwareupdate_tab.php

```diff
@@ -0,0 +1,168 @@
+					<th data-i18n="softwareupdate.automaticdownload"></th>
+					<td id="softwareupdate-automaticdownload"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.configdatainstall"></th>
+					<td id="softwareupdate-configdatainstall"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.criticalupdateinstall"></th>
+					<td id="softwareupdate-criticalupdateinstall"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.mrxprotect"></th>
+					<td id="softwareupdate-mrxprotect"></td>
+				</tr>	
+				<tr>
+					<th data-i18n="softwareupdate.lastbackgroundccdsuccessfuldate"></th>
+					<td id="softwareupdate-lastbackgroundccdsuccessfuldate"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.lastbackgroundsuccessfuldate"></th>
+					<td id="softwareupdate-lastbackgroundsuccessfuldate"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.lastfullsuccessfuldate"></th>
+					<td id="softwareupdate-lastfullsuccessfuldate"></td>
+				</tr>
+                				<tr>
+					<th data-i18n="softwareupdate.lastsuccessfuldate"></th>
+					<td id="softwareupdate-lastsuccessfuldate"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.lastresultcode"></th>
+					<td id="softwareupdate-lastresultcode"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.lastsessionsuccessful"></th>
+					<td id="softwareupdate-lastsessionsuccessful"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.lastrecommendedupdatesavailable"></th>
+					<td id="softwareupdate-lastrecommendedupdatesavailable"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.lastupdatesavailable"></th>
+					<td id="softwareupdate-lastupdatesavailable"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.recommendedupdates"></th>
+					<td id="softwareupdate-recommendedupdates"></td>
+				</tr>
+                <tr>
+					<th data-i18n="softwareupdate.inactiveupdates"></th>
+					<td id="softwareupdate-inactiveupdates"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.skiplocalcdn"></th>
+					<td id="softwareupdate-skiplocalcdn"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.lastattemptsystemversion"></th>
+					<td id="softwareupdate-lastattemptsystemversion"></td>
+				</tr>
+				<tr>
+					<th data-i18n="softwareupdate.catalogurl"></th>
+					<td id="softwareupdate-catalogurl"></td>
+				</tr>
+			</table>
+		</div>
+	</div>
+
+<script>
+$(document).on('appReady', function(e, lang) {
+	
+	// Get softwareupdate data
+	$.getJSON( appUrl + '/module/softwareupdate/get_data/' + serialNumber, function( data ) {
+		if( ! data.mrxprotect){
+			$('#softwareupdate-msg').text(i18n.t('no_data'));
+		}
+		else{
+			
+			// Hide
+			$('#softwareupdate-msg').text('');
+			$('#softwareupdate-view').removeClass('hide');
+			
+			// Add strings
+			if(data.automaticcheckenabled === "1" || data.automaticcheckenabled === 1) {
+				 $('#softwareupdate-automaticcheckenabled').text("Yes");
```

Comment:
I tried, but I wasn't able to find out how to make the localization framework work in JavaScript like that